cfg/context.go

```diff
@@ -0,0 +1,224 @@
+// Copyright 2022 The Drivers Cooperative. All rights reserved.
+// Use of this source code is governed by a dual
+// license that can be found in the LICENSE file.
+
+package cfg
```

Comment:
package cfg implies containing something to do with config, in this regard cfg.context is misleading.

Reply:
"conf" package will be retired after lightweight configuration will be implemented. All new configuration logic will be put in "cfg" package. "conf" is too long and does not follow existing naming such as "api", "log".

Reply:
i meant maybe context had to be put under 'context' named package, not cfg, config, conf, etc. matter of taste i guess. not that much important.

Reply:
I though about this. Putting it in the "context" will slash with the std lib context package. Not a good idea. "cfg" is the most convenient place since it will be used almost everywhere either for config vars or context.